lib/app.js

```diff
@@ -404,17 +405,20 @@ module.exports = {
             return value;
           }
         }
-      }];
-
-      getPrompt(argv, schema, function(results) {
-        if (path.relative('.', results.public).match(/^\./)) {
-          console.log(chalk.red('init cancelled - the public directory must be within the current working directory'));
-          process.exit(1);
+      };
+      schemaPublic.conform = function(value) {
+        if (path.relative('.', value).match(/^\./)) {
+          schemaPublic.message = 'Please use a sub directory of current working directory';
```

Comment:
reasoning: I don't like 'sub' on its own (it should be hyphenated at least) but the terminology is perhaps a little technical. People understand directories in other directories better than 'sub-directories' I believe